controllers/pattern_controller.go

```diff
@@ -68,6 +68,10 @@ type PatternReconciler struct {
 //+kubebuilder:rbac:groups=config.openshift.io,resources=ingresses,verbs=list;get
 //+kubebuilder:rbac:groups=config.openshift.io,resources=infrastructures,verbs=list;get
 //+kubebuilder:rbac:groups="",resources=namespaces,verbs=list;get
+//FIXME(bandini): it would be nice to restrict the following two RBACs to the vault namespace. kustomize currently fails on that
+//+kubebuilder:rbac:groups="",resources=pods,verbs=list;get
+//+kubebuilder:rbac:groups="",resources=pods/exec,resourceNames=vault-0,verbs=create
```

Comment:
Is there any way that the pod wouldn't be called vault-0 ?

Reply:
So by default we install vault in standalone mode (aka non-HA running in a single pod called vault-0). Let me explore the HA non-standalone variant and see how things look in that case

Reply:
So I looked at this a bit more, and all our stock vault installation are in standalone mode (aka 1 pod and 1 pv). To install vault in a different mode (HA), there would be some work required in the pattern as most things won't work out of the box in HA mode.

Reply:
ok